### PR TITLE
No ticket/skeleton service first draft

### PR DESCRIPTION
**Pull request by awhodgson256**
Add test endpoint, fix build and draft initial helm deploy / charts

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/findandreferanintervention/controller/HelloController.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.hmppsfindandreferaninterventionsservice.controller
+
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@PreAuthorize("hasRole('ROLE_PROBATION') or hasRole('ROLE_CRS_PROVIDER') or hasRole('ROLE_INTERVENTIONS_API_READ_ALL')")
```
Comment (amolphalnikar) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/findandreferanintervention/controller/HelloController.kt:8]: Don't think we need PreAuthorize for testing this API. Can be commented out.

Comment (amolphalnikar) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/findandreferanintervention/controller/HelloController.kt:8]: Or can just keep ROLE_INTERVENTIONS_API_READ_ALL

File: helm_deploy/hmpps-find-and-refer-an-intervention-service/values.yaml [hunk 2]
```diff
@@ -1,18 +1,18 @@
 generic-service:
-  nameOverride: hmpps-template-kotlin
-  productId: "UNASSIGNED" # productId for the product that this belongs too, i.e. DPS001, see README.md for details
+  nameOverride: hmpps-find-and-refer-an-intervention-service
+  productId: "HMPPS545" # productId for the product that this belongs too, i.e. DPS001, see README.md for details
 
   replicaCount: 4
```
Comment (amolphalnikar) [inline helm_deploy/hmpps-find-and-refer-an-intervention-service/values.yaml:5]: Can we set replicaCount to 2 to start with ?

**Review by amolphalnikar — COMMENTED** [review]
I left couple of comments. Please check.

File: helm_deploy/hmpps-find-and-refer-an-intervention-service/values.yaml (hunk 2 above)
Comment (awhodgson256) [reply helm_deploy/hmpps-find-and-refer-an-intervention-service/values.yaml]: Done

**Review by amolphalnikar — APPROVED** [review]

**Pull request merged**